geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9990,6 +10011,43 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  PartitionRegionConfig updatePRNodeInformation() {
+    if (cache.getDistributedSystem().getProperties().get(ENABLE_CLUSTER_CONFIGURATION) != "true") {
+      return null;
+    }
```

Comment:
Why does this updatePRNodeInformation method do nothing if cluster configuration is not enabled? It seems like we need to make sure this metadata is consistent no matter what, right?

Reply:
The main reason we added this stipulation is so we don't disrupt any clients who have workflows set up around defining the region with the original definition and re-mutating the region on each new member. That workflow would already be disrupted if the user were to use cluster config, as cluster config would attempt to define the region based on it's cluster configuration definition after the loader/writer was added.

If we don't think that would be an issue for users, then we could definitely remove it.

Reply:
We had a discussion about this and decided it would be better to remove this restriction on the update. The failure to update prRoot resulting in validation failures is no intended behavior and we don't believe we have customers relying on that behavior, so it's better to have the fix work for all Geode configuration methods and not just cluster configuration.

Reply:
Changed to always update PR node information 
regardless if cluster config is enabled.